components/WarningNote/WarningNoteRecap/WarningNoteRecap.spec.tsx

```diff
@@ -2,25 +2,47 @@ import { render } from '@testing-library/react';
 
 import WarningNoteRecap from './WarningNoteRecap';
 import * as warningNotes from 'utils/api/warningNotes';
-import { mockedWarningNote } from 'factories/warningNotes';
+import { mockedWarningNote, warningNoteFactory } from 'factories/warningNotes';
 import { mockedResident } from 'factories/residents';
 
 jest.mock('components/Spinner/Spinner', () => () => 'MockedSpinner');
 
 describe(`WarningNoteRecap`, () => {
-  jest.spyOn(warningNotes, 'useWarningNote').mockImplementation(() => ({
-    data: mockedWarningNote.find(({ id }) => id === props.warningNoteId),
-    mutate: jest.fn(),
-    revalidate: jest.fn(),
-    isValidating: false,
-  }));
   const props = {
     person: mockedResident,
     warningNoteId: 1,
   };
 
+  beforeEach(() => {
+    jest.resetAllMocks();
```

Comment:
Is this kind of like an initial teardown before running the tests? 🤔

Reply:
Yeah you're absolutely right. This a force of habit on my behalf, muscle memory I've always just used `beforeEach` 😅 . 

It effectively says:
>before each test, reset the mocks back to their default state

You could also do it in an `afterEach` and it'd basically do the same thing